test/bats/aws.bats

```diff
@@ -145,6 +290,6 @@ teardown_file() {
   assert_success
 }
 
-teardown_file() {
-  archive_info || true
-}
+# teardown_file() {
+#   archive_info || true
+# }
```

Comment:
We have commented this out because the previously defined `teardown_file()` is getting overwritten.